src/containers/org/DataSetDataContainer.js

```diff
@@ -70,10 +69,11 @@ const DataSetDataContainer = ({
       searchHits.forEach((entity :Map) => mutableSet.union(entity.keySeq()));
     });
     const headers :List = dataSetColumns
-      .filter((column :Map<FQN, List>) => headersSet.has(getPropertyValue(column, [FQNS.OL_TYPE, 0])))
-      .map((column :Map<FQN, List>) => {
-        const fqn :string = getPropertyValue(column, [FQNS.OL_TYPE, 0]);
-        const title :string = getPropertyValue(column, [FQNS.OL_TITLE, 0]);
+      .valueSeq()
+      .filter((column :Map) => headersSet.has(column.get('name')))
+      .map((column :Map) => {
+        const fqn :string = column.get('name');
+        const title :string = column.getIn(['metadata', 'title']);
```

Comment:
should we pull `metadata`, `name`, and `title` into constants somewhere?

Reply:
Yes. I'll do that after merging `develop` and fixing the conflicts.